docs/source/RunQemuUnittests.rst

```diff
@@ -1,6 +1,48 @@
-======================================
-Running QEMU unittests with Avocado-VT
-======================================
+===========================
+Running QEMU kvm-unit-tests
+===========================
+
+Currently there are two ways to run kvm-unit-tests. Newer one with per test
+results method is `Run kvm-unit-test in avocado`_ and the older one
```

Comment:
Hummm... I got that from the repo name and from the README file:

```
Welcome to kvm-unit-tests

See http://www.linux-kvm.org/page/KVM-unit-tests
```

And of course from my memories interacting with it... but things change, and names change even more, so if you know for a fact that this is the "new name" let's use it.